### PR TITLE
fix wpcs in plugin generation

### PR DESCRIPTION
**Pull request by thrijith**
update code to remove WPCS warnings in plugin generated files  #109

### CODE REVIEW

**Comment by gitlost** [comment]
Thanks very much for the PRs @thrijith. I'm going to remove https://github.com/wp-cli/scaffold-command/issues/109 from the 1.1.12 milestone as incomplete at this late stage, so will be reviewing soon after WP-CLI 1.5.0 releases....

File: templates/plugin-bootstrap.mustache [hunk 1]
```diff
@@ -12,7 +12,7 @@ if ( ! $_tests_dir ) {
 }
 
 if ( ! file_exists( $_tests_dir . '/includes/functions.php' ) ) {
-	echo "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL;
+	esc_html( "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" ) . PHP_EOL;
```
Comment (gitlost) [inline templates/plugin-bootstrap.mustache]: Still need the `echo` here but from what I can see stuff in the `tests` directory is ignored so don't need this change at all??

File: templates/plugin-test-sample.mustache [hunk 2]
```diff
@@ -13,7 +13,7 @@ class SampleTest extends WP_UnitTestCase {
 	/**
 	 * A single example test.
 	 */
-	function test_sample() {
+	public function test_sample() {
```
Comment (gitlost) [inline templates/plugin-test-sample.mustache]: Ditto stuff in the tests directory is ignored so don't think this change is needed.

**Comment by gitlost** [comment]
@thrijith in a bit of a rush (read panic) so took the liberty of removing the tests directory changes. Will revisit after 1.5.0, ta v much for the PR!

File: templates/plugin-test-sample.mustache (hunk 2 above)
Comment (thrijith) [reply templates/plugin-test-sample.mustache]: https://wordpress.slack.com/archives/C02RP4T41/p1516728175000577 
that's why I asked 'coz I was not sure if these needed changes

Comment (gitlost) [reply templates/plugin-test-sample.mustache]: Ah I see sorry, so were you getting errors on running phpcs on stuff in the tests directory?

**Pull request merged**